modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/inputs/IntegerInput.tsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClaySelectWithOption} from '@clayui/form';
+import React from 'react';
+
+
+interface Props {
+    disabled?: boolean;
+    onChange: (payload: {value: string}) => void;
+    options: [
+        {
+            disabled: boolean,
+            label: string,
+            value: string,
+        }
+    ]
```

Comment:
If we want to specify an Array of elements, it should be either:

```
{ disabled: booolean, /** ... **/ }[]
```

or


```
Array<{ disabled: booolean, /** ... **/ }>
```

both of them are valid. I tend to use `Array<>` for multiple line expressions like this one, and `*[]` for single line ones, but it really does not matter.

However, if we write something like `[{ disabled: boolean }]`, we are specifying that we want an array with just one element, which I think it's not the case :sweat_smile:

Reply:
Understood, thank u!